python/tchannel/zipkin/zipkin_trace.py

```diff
@@ -87,7 +87,7 @@ def after_receive_error(self, error):
         error.tracing.annotations.append(ann)
         self.tracer.record([(error.tracing, error.tracing.annotations)])
 
-    def after_send_error(self, error):
+    def after_send_system_error(self, error):
```

Comment:
Now I am only update the outbound events now. For others, will be updated in the inbound events.